src/service/resource.js

```diff
@@ -488,6 +488,7 @@ export class Resource {
       this.premeasuredRect_ = null;
     } else {
       this.computeMeasurements_();
+      this.premeasuredRect_ = null;
```

Comment:
Let's move this down, so we can reduce the duplication in both branches?

Reply:
Done